src/Selenium2Library/keywords/_screenshot.py

```diff
@@ -19,14 +19,22 @@ def capture_page_screenshot(self, filename=None):
         the Robot Framework log file is written into. The `filename` is
         also considered relative to the same directory, if it is not
         given in absolute format. If an absolute or relative path is given
-        but the path does not exist it will be created. 
+        but the path does not exist it will be created.
 
-        `css` can be used to modify how the screenshot is taken. By default
-        the bakground color is changed to avoid possible problems with
-        background leaking when the page layout is somehow broken.
-        """
-        path, link = self._get_screenshot_paths(filename)
+        `index` with index argument it is possible to define custom filename
+        but each time get unique screen capture. Example if pabot is used
+        to run multiple test suites in paraller and correct screenshots should
+        be copied to log, then it is possible to use `filename` and
+        `index` argument to gether to create unique name for each screenshot.
 
```

Comment:
That's close to what I was thinking. The only thing about that I'm not that fond of is the fact that there's a global counter. I would much rather each file have it's own count, though I know that would be a little bit more work. 

Essentially the flow would be like this.

```python
# This code is more psudo than an actual implementation
if not overwrite:
  files = listdir(path)
  file = filename
  while file in files:
    file = filename + '-' + index + extension
    index += 1
# write file
```

That's only slightly more complicated, but it works more constantly to how I would expect it to. 

Your test would then be like this

```robotframework
| Screen Capture | filename=${BROWSER}.png |
| Screen Capture | filename=${BROWSER}.png |
| File Should Exist  | ${OUTPUTDIR}${/}${BROWSER}.png |
| File Should Exist  | ${OUTPUTDIR}${/}${BROWSER}-1.png |
| Screen Capture |
| Screen Capture |
| File Should Exist  | ${OUTPUTDIR}${/}selenium-screenshot-1.png |
| File Should Exist  | ${OUTPUTDIR}${/}selenium-screenshot-2.png |
| Screen Capture | filename=overwrite-${BROWSER}.png | overwrite=${True} |
| Screen Capture | filename=overwrite-${BROWSER}.png | overwrite=${True} |
| File Should Exist  | ${OUTPUTDIR}${/}overwrite-${BROWSER}.png | 
| File Should Not Exist | ${OUTPUTDIR}${/}overwrite-${BROWSER}-1.png | 
```